web/app/scripts/toddow/toddow-directive.js

```diff
@@ -148,6 +147,6 @@
     }
 
     angular.module('driver.toddow')
-    .directive('driverToddow', ToDDoW);
+      .directive('driverToddow', ToDDoW);
```

Comment:
We have these two lines aligned in all the other files; might as well keep it consistent.